timemachine/fe/topology.py

```diff
@@ -329,6 +331,38 @@ def parameterize_periodic_torsion(self, proper_params, improper_params):
         combined_potential = potentials.PeriodicTorsion(combined_idxs, combined_lambda_mult, combined_lambda_offset)
         return combined_params, combined_potential
 
+    def setup_chiral_restraints(self):
+        # setup chiral restraints
+        chiral_atoms = chiral_utils.find_chiral_atoms(self.mol)
+        chiral_bonds = chiral_utils.find_chiral_bonds(self.mol)
+
+        restraint_k = [1000.0]
```

Comment:
Would it be useful to expose `chiral_atom_restraint_k=1000.0`, `chiral_bond_restraint_k=1000.0` as parameters in `setup_chiral_restraints(self)` (or add them as attributes)? (Not sure if we'd ever want to modify them separately.)

Reply:
this seems reasonable! willfix